relay-server/src/metrics_extraction/event.rs

```diff
@@ -1559,8 +1526,8 @@ mod tests {
                 "ttfd": "ttfd"
             }
         }"#;
-        let span = Annotated::from_json(span).unwrap().into_value().unwrap();
-        let metrics = extract_span_metrics(&span);
+        let span: Span = Annotated::from_json(span).unwrap().into_value().unwrap();
```

Comment:
nit: was it a best practice to use the turbofish operator? I can't find any references:
```suggestion
        let span = Annotated::<Span>::from_json(span).unwrap().into_value().unwrap();
```

Reply:
👍 Maybe not a best practice, but it's more explicit and it makes this call consistent with the one below.